trunk/NDHMS/Routing/module_noah_chan_param_init_rt.F

```diff
@@ -72,13 +73,15 @@ subroutine CHAN_PARM_INIT (BOTWID, HLINK_INIT, CHAN_SS, CHMann)
 #endif
        do ORDER=1,CHANCATS
 
-          read (IUNIT,*)IINDEX, BOTWID(ORDER), HLINK_INIT(ORDER), CHAN_SS(ORDER), CHMann(ORDER)
+          read (IUNIT,*)IINDEX, BOTWID(ORDER),CHANN_K(ORDER),HLINK_INIT(ORDER), CHAN_SS(ORDER),  &
```

Comment:
I am not sure if this is useful. I never tested ChannK in a .TBL file. This theoretically could be added later or activated with a namelist option in the future.